ipaserver/install/certs.py

```diff
@@ -85,6 +85,23 @@ def export_pem_p12(pkcs12_fname, pkcs12_pwd_fname, nickname, pem_fname):
                  "-passout", "file:" + pkcs12_pwd_fname])
 
 
+def strip_csr_header(csr):
+    """
```

Comment:
I did not find a place where this function is used.

Reply:
Ah, you're right, I removed it but it got back in a rebase accident, I will remove it.